src/common-rest-api/src/main/java/org/niis/xroad/restapi/util/RequestHelper.java

```diff
@@ -96,7 +98,23 @@ public void runInRequestScope(RequestScopeVoidOperation operation) {
     public interface RequestScopeOperation {
         Object executeInRequest();
     }
+
     public interface RequestScopeVoidOperation {
         void executeInRequest();
     }
+
+    /**
+     * Gets request senders IP address
+     */
+    public String getRequestSenderIPAddress() {
+        String ipAddress = null;
+        final HttpServletRequest currentHttpRequest = getCurrentHttpRequest();
+        if (currentHttpRequest != null) {
+            ipAddress = currentHttpRequest.getHeader(X_FORWARDED_FOR);
```

Comment:
Same thing, just:
```java
ipAddress = currentHttpRequest.getRemoteAddr();
```
is sufficient when `server.forward-headers-strategy=native` is added to admin service's `application.yml`

Reply:
yes, it is sufficient currently. But the mentioned classes might be used from other modules with different configuration. We do not have it right now, but I wanted to be on the safe side here - it should work when headers are still present.